pyseir/inference/whitelist_generator.py

```diff
@@ -5,7 +5,7 @@
 from datetime import datetime
 from pyseir.utils import get_run_artifact_path, RunArtifact
 from pandarallel import pandarallel
-pandarallel.initialize(progress_bar=True)
+pandarallel.initialize(progress_bar=False)
```

Comment:
Read from env? lie `PYSEIR_VERBOSITY`?  I like the progress bars on local.

Reply:
good to know